spec/factories.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
+  factory :strategy_email_reminder do
+    
```

Comment:
yeah thats true will remove it, it was auto generated when i did `rails g model`